torch/csrc/distributed/rpc/init.cpp

```diff
@@ -338,20 +338,32 @@ If the future completes with an error, an exception is thrown.
   module.def(
       "_invoke_rpc_torchscript",
       [](const std::string& dstWorkerName,
-         const std::string& qualifiedNameStr,
-         const py::args& args,
-         const py::kwargs& kwargs) {
+         const py::object& userFunctionPyObj,
+         const py::tuple& argsTuple,
+         const py::dict& kwargsDict) {
```

Comment:
why does it have to be py::tuple and py::dict?

Reply:
Because `py::args` takes unrolled args, it takes `function(arg1, arg2)`, and that will be rolled into `py::args args = (arg1, arg2)`.

but I want to pass in `function(args=(arg1, arg2))`, and that we will have `py::tuple args = (arg1, arg2)`.